source/Ship.cpp

```diff
@@ -1781,6 +1781,22 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 			if(energy < cost * fabs(commands.Turn()))
 				commands.SetTurn(commands.Turn() * energy / (cost * fabs(commands.Turn())));
 
+			cost = attributes.Get("turning shields");
+			if(shields < cost * fabs(commands.Turn()))
+				commands.SetTurn(commands.Turn() * shields / (cost * fabs(commands.Turn())));
+
+			cost = attributes.Get("turning hull");
+			if(hull < cost * fabs(commands.Turn()))
+				commands.SetTurn(commands.Turn() * hull / (cost * fabs(commands.Turn())));
+
+			cost = attributes.Get("turning fuel");
+			if(hull < cost * fabs(commands.Turn()))
```

Comment:
```suggestion
			cost = attributes.Get("turning fuel");
			if(fuel < cost * fabs(commands.Turn()))
```
?

Reply:
Pretty sure I even spotted that one in the past too and completely forgot about it.